RubberduckTests/QuickFixes/RemoveDuplicatedAnnotationQuickFixTests.cs

```diff
@@ -0,0 +1,200 @@
+        {
+            const string inputCode = @"
+'@Obsolete: Foo
+'@Obsolete
+Public Sub Foo
+End Sub";
+
+            const string expectedCode = @"
+': Foo
+'@Obsolete
+Public Sub Foo
+End Sub";
+
+            var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out var component);
+            using (var state = MockParser.CreateAndParse(vbe.Object))
+            {
+                var inspection = new DuplicatedAnnotationInspection(state);
+                var inspectionResults = inspection.GetInspectionResults(CancellationToken.None);
+
+                new RemoveDuplicatedAnnotationQuickFix(state).Fix(inspectionResults.First());
+                Assert.AreEqual(expectedCode, state.GetRewriter(component).GetText());
+            }
+        }
+
+        [Test]
+        [Category("QuickFixes")]
+        public void RemoveDuplicatedAnnotation_QuickFixWorks_RemoveDuplicateFromSameAnnotationList()
+        {
+            const string inputCode = @"
+'@Obsolete @Obsolete
+Public Sub Foo
+End Sub";
+
+            const string expectedCode = @"
+'@Obsolete 
+Public Sub Foo
+End Sub";
+
+            var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out var component);
+            using (var state = MockParser.CreateAndParse(vbe.Object))
+            {
+                var inspection = new DuplicatedAnnotationInspection(state);
+                var inspectionResults = inspection.GetInspectionResults(CancellationToken.None);
+
+                new RemoveDuplicatedAnnotationQuickFix(state).Fix(inspectionResults.First());
+                Assert.AreEqual(expectedCode, state.GetRewriter(component).GetText());
+            }
+        }
+
+        [Test]
+        [Category("QuickFixes")]
+        public void RemoveDuplicatedAnnotation_QuickFixWorks_RemoveMultipleDuplicatesFromSameAnnotationList()
+        {
+            const string inputCode = @"
+'@Obsolete @Obsolete @Obsolete
+Public Sub Foo
+End Sub";
+
+            const string expectedCode = @"
+'@Obsolete  
+Public Sub Foo
+End Sub";
+
+            var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out var component);
+            using (var state = MockParser.CreateAndParse(vbe.Object))
+            {
+                var inspection = new DuplicatedAnnotationInspection(state);
+                var inspectionResults = inspection.GetInspectionResults(CancellationToken.None);
+
+                new RemoveDuplicatedAnnotationQuickFix(state).Fix(inspectionResults.First());
+                Assert.AreEqual(expectedCode, state.GetRewriter(component).GetText());
+            }
+        }
+
+        [Test]
+        [Category("QuickFixes")]
+        public void RemoveDuplicatedAnnotation_QuickFixWorks_RemoveDuplicateFromOtherAnnotationList()
+        {
+            const string inputCode = @"
+'@Obsolete @NoIndent
+'@Obsolete
+Public Sub Foo
+End Sub";
+
+            const string expectedCode = @"
+' @NoIndent
```

Comment:
hmm .. I'd want to remove this whitespace. Not sure `'@NoIndent` is recognized like that, considering that the grammar rule starts `annotationList : SINGLEQUOTE (AT annotation whiteSpace?)+[..]`. Note the lack of permitted whitespace after the `SINGLEQUOTE`

Reply:
I think this can be fixed by adding `rewriter.Remove(annotationList.whiteSpace(index));`, but I'm not definitively sure

Reply:
I will try to correct this.